server_injected.Dockerfile

```diff
@@ -15,9 +19,13 @@ LABEL io.parity.image.authors="cicd-team@parity.io" \
     io.parity.image.created="${BUILD_DATE}"
 
 WORKDIR /backend
+
+COPY ./package.json .
+COPY ./yarn.lock .
+RUN yarn --network-concurrency 1 --frozen-lockfile
+
 COPY . .
-# uncomment to fix build on MacOS Apple Silicon chip
-# RUN apk add --no-cache python3 make g++
-RUN apk add git==2.30.6-r0
+RUN yarn prepare && yarn build
```

Comment:
Wouldn't be better to change the `prepare` script to `prebuild` so it's done automatically?

Reply:
Sounds good, changed ✅